embassy-stm32/src/gpio.rs

```diff
@@ -148,6 +184,18 @@ impl<'d, T: Pin> StatefulOutputPin for Output<'d, T> {
     }
 }
 
+impl<'d, T: Pin> InputPin for Output<'d, T> {
+    type Error = Infallible;
+
+    fn is_high(&self) -> Result<bool, Self::Error> {
+        self.is_set_high()
```

Comment:
`is_set_high/is_set_low` return the output value we've set, not the input from outside the mcu, that's probably not what you want?

Reply:
Yeah, just noticed it reads from `odr` not `idr`.